frontend/src/metabase-types/api/mocks/settings.ts

```diff
@@ -125,6 +125,7 @@ export const createMockTokenFeatures = (
   serialization: false,
   cache_preemptive: false,
   database_routing: false,
+  "development-mode": false,
```

Comment:
This is that thing where we have `development-mode?` in settings and `development-mode` as a token feature. Definitely won't confuse anyone in the future :stuck_out_tongue_winking_eye:

Is there something we can do about it? Does it need to be set in settings if it's on the token?

Reply:
Yeah, I saw that we're not super consistent in general with how we grab these things. I'm not sure if this is new or not, but it looks like most (if not all) token features have associated settings that are used to derive them. We have tooling to look directly at token features, but maybe we should be looking at the settings instead? I'm not sure

Reply:
Yeah totally existing problem. I think the backend just reads the token features and puts them into settings. If we were going to go one way, I would have said we keep token features out of settings (I mean token features itself is in settings, but not splat the individual features onto settings as well)